05.wc/wc.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, filenames = parse_options
+  file_stats = collect_file_stats(filenames)
+  total_stats = calculate_total_stats(file_stats)
+  max_widths = calculate_max_widths(total_stats)
+  print_file_stats(file_stats, max_widths, options)
+  print_file_stats([total_stats], max_widths, options) if filenames.size > 1
```

Comment:
```suggestion
  file_stats.each do |file_stat|
    print_file_stats(file_stat, max_widths, options)
  end
  print_file_stats(total_stats, max_widths, options) if filenames.size > 1
```

些細な違いですが、`[total_stats]`のようにわざわざtotalを配列にするのがちょっと不自然なので、それなら1件ずつ表示するメソッドにした方がスマートな気がしました。